### ISSUE TITLE
IS DIS THE MAKFN UNDEDECTED SOFT AIM HAK OMG

### ISSUE THREAD

**Issue by Xyrem** [PastoThePaster/MakFN-Spoofer-Src#1]
HAKOR!!!!!!!!!1

**Comment by Micca1338** [comment]
HOLY SHIOTYN {"
AS{PDHO"asl;dj/akmsd,sd[
p;as\

**Comment by PastoThePaster** [comment]
> HAKOR!!!!!!!!!1

NO MEN NO HAKOR

**Comment by ghost** [comment]
Yes big hackor he bought the source to leak lmao. That's fine by me, do whatever you want with your money 🤷‍♂️

**Comment by Micca1338** [comment]
Exposed smh

**Comment by Limitlessssss** [comment]
MORE PASTED FN SOURCESSS LETS GOOOOOOOOOOO LETS SEE IF ANYBODY CAN EVEN CREATE AN UNDETCTED INJECTOR FOR EAC/BE USING KERNEL JK LOVE U PASTERS ;)
